cmd/handler-utils.go

```diff
@@ -114,16 +114,24 @@ func extractMetadataFromHeader(header http.Header) (map[string]string, error) {
 			metadata[supportedHeader] = header.Get(supportedHeader)
 		}
 	}
+	const MaxHeaderSize = 8 * 1024
```

Comment:
We should define the constants here.. ideally these should be global. With more descriptive naming 

```
putObjectMaxHeaderSize   = 8 * humanize.KiByte
putObjectMaxUserDataSize = 2 * humanize.KiByte
```